XcodeServerSDKTests/RepositoryTests.swift

```diff
@@ -79,5 +79,35 @@ class RepositoryTests: XCTestCase {
         sshEnum = .LoggedInReadWrite
         XCTAssertEqual(sshEnum.toString(), "All logged in users can read and write")
     }
+    
+    // MARK: API Routes tests
+    func testGetRepositories() {
+        let expectation = self.expectationWithDescription("Get Repositories")
+        let server = self.getRecordingXcodeServer("get_repositories")
+        
+        server.getRepositories() { (repositories, error) in
+            XCTAssertNil(error, "Error should be nil")
+            XCTAssertNotNil(repositories, "Repositories shouldn't be nil")
+            
+            if let repos = repositories {
+                XCTAssertEqual(repos.count, 2, "There should be two repositories available")
+                
+                for (index, repo) in repos.enumerate() {
+                    XCTAssertEqual(repo.name, "Test\(index + 1)")
```

Comment:
Hahaha, I get your point correctly 😄 I was just overthinking and looking far far ahead and drawing scenarios for _future_ possibilities.

To sum up, I write test, I create it based on my Xcode Server configuration and write tests for this - I don't give a shit what's on your Xcode Server, right?

**DVR** is cool but I'd be better if all of use use the same instance/configuration of Xcode Server so if any `json` casette get lost/removed we can easily revert the state back without revriting tests or modifying something on OS X Server - you get my point?